Grid/Grid.php

```diff
@@ -911,7 +912,7 @@ protected function processPage($page, $filtering = false)
     protected function processOrder($order)
     {
         if ($order !== null) {
-            list($columnId, $columnOrder) = explode('|', $order);
+            [$columnId, $columnOrder] = explode('|', $order);
```

Comment:
Questa modifica e la semplificazione degli if le ha fatte PHPStorm in automatico :thinking: